docs/linux.md

```diff
@@ -49,16 +50,18 @@ User=ollama
 Group=ollama
 Restart=always
 RestartSec=3
+Environment="PATH=$PATH"
```

Comment:
Do we need this line?
```suggestion
```

Reply:
I think we do, see

https://github.com/ollama/ollama/blob/94fff5805fecd25287e1a21ac9910859011adcff/scripts/install.sh#L134